### PR TITLE
refactor: install next

### PR DESCRIPTION
**Pull request by waynewee**
## Summary

Set up nextjs

## Changes

add next commands
updated postcss file
created styles and pages folder
moved tailwind.css to styles/globals.css
created _app.tsx and _document.tsx

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,10 @@
     "test:coverage": "npm run test -- --collectCoverage",
     "test:watch": "npm run test -- --watchAll",
     "test:update": "npm run test -- -u",
-    "wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:3000"
+    "wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:3000",
+    "next-dev": "next dev",
```
Comment (Nebulis) [inline package.json:38]: end goal is to replace the other scripts right ?

File: pages/_app.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { configureStore } from "../src/store";
+import Head from "next/head";
+
+const store = configureStore();
+
+export default function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (Nebulis) [inline pages/_app.tsx:20]: why isnt the head portion in _document?

File: pages/eta.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (Nebulis) [inline pages/eta.tsx:1]: what's this page about?

File: pages/event.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (Nebulis) [inline pages/event.tsx:1]: what's this page about?

File: pages/eta.tsx (hunk 3 above)
Comment (Nebulis) [reply pages/eta.tsx:1]: ok found :)

File: pages/event.tsx (hunk 4 above)
Comment (Nebulis) [reply pages/event.tsx:1]: ok found :)

File: pages/_app.tsx (hunk 2 above)
Comment (waynewee) [reply pages/_app.tsx:20]: The viewport and title tags cannot be placed in _document. See [here](https://stackoverflow.com/questions/65832820/next-js-viewport-meta-tags-should-not-be-used-in-document-jss-head)

File: package.json (hunk 1 above)
Comment (waynewee) [reply package.json:38]: yeap this is just for now

File: pages/_app.tsx (hunk 2 above)
Comment (Nebulis) [reply pages/_app.tsx:20]: ah ok interesting, didnt remember this 👍

File: src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx [hunk 5]
```diff
@@ -85,11 +85,11 @@ export const DemoCreateIssue: FunctionComponent = () => {
               <h3>Failed</h3>
               <p className="py-5">
                 Please check if you have internet connection,
-                <br /> alternatively, see <Link to="/faq">FAQ</Link> or <Link to="/contact">Contact us</Link>
+                <br /> alternatively, see <Link href="/faq">FAQ</Link> or <Link href="/contact">Contact us</Link>
```
Comment (Nebulis) [inline src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx]: missing `<a>` inside `<Link>` ?

File: src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx [hunk 6]
```diff
@@ -74,7 +74,7 @@ export const DemoCreateIssue: FunctionComponent = () => {
                 </a>
                 .
               </p>
-              <Link to="/demo/verify">
+              <Link href="/demo/verify">
```
Comment (Nebulis) [inline src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx:77]: hum button is not a link? according to the doc if it's a link must use passRef and eventually forward ref

Also a11y wise, button should be alink

**Pull request closed without merging**